src/components/Filters/Filters.tsx

```diff
@@ -80,10 +67,14 @@ function Filters<F, G extends Value = string>(props: FilterProps<F, G>) {
   return (
     <div css={Css.df.itemsCenter.childGap1.$} {...testId}>
       {maybeGroupByField}
-      {pageFilters.map((c, idx) => (
-        <div key={idx}>{c}</div>
+
+      {safeEntries(pageFilters).map(([key, f]: [keyof F, Filter<any>]) => (
+        <div key={key as string}>
+          {f.render(filter[key], (value) => onChange(updateFilter(filter, key, value)), testId, false)}
```

Comment:
Aha! I was trying to figure out where/how is the Filter class's `render` method being called with these props? When I first saw the classes, I assumed it was a react Class Component, but now I realize it is just a generic class. That makes more sense.

Reply:
Oh yeah, that is confusing. :-) At first I had `renderInPage` and `renderInModal` as separate methods, so it was more obviously "not a react component"; but for most filters, those are ~90% the same logic, so I changed it to a single `render` method with a `modal` prop...

Would not be against a "not-`render`" name for that method...ideas?

Reply:
`render` makes too much sense to rename. Maybe a comment above the class definition saying "Not a react class component"? I dunno, probably should be easy to realize as it doesn't `extend React.Component`